frontend/src/app/services/lessons.service.ts

```diff
@@ -35,4 +35,25 @@ export class LessonsService {
     getQuestions(id: number) {
         return this.http.get(`${this.routePrefix}/${id}/questions`);
     }
+
+    getTeacherStatistics() {
+        return this.http.get(`${this.routePrefix}/statistics`);
+    }
+
+    getTeacherLessonsAtPeriod(start: string, end: string) {
+        return this.http.get(`${this.routePrefix}/${start}/${end}`);
+    }
+
+    addTimeOffset(date: string): string {
+        const offset = new Date().getTimezoneOffset();
+        const dateObject = new Date(date);
+
+        dateObject.setMinutes(dateObject.getMinutes() - offset);
+
+        return dateObject.toString();
+    }
```

Comment:
It's bettter to move such parts to helpers, as services are more working with data, like data, you get from server